libs/feature/search/src/lib/fuzzy-search/fuzzy-search.component.ts

```diff
@@ -76,6 +79,8 @@ export class FuzzySearchComponent implements OnInit {
   }
 
   handleInputCleared() {
-    this.searchService.updateFilters({ any: '' })
+    if (this.currentSearchFilters.any) {
```

Comment:
Instead of storing an internal state here you can do something like so:
```ts
async handleInputCleared() {
  const currentSearchFilters = await firstValueFrom(this.searchFacade.searchFilters$);
  if (!currentSearchFilters.any) { return; }
  this.searchService.updateFilters({ any: '' })
}
```

Reply:
Nice idea, also works as expected, thanks! I just inverted the `if` to save another line.